recipes/harfbuzz/all/conanfile.py

```diff
@@ -29,7 +29,7 @@ class HarfbuzzConan(ConanFile):
         "with_glib": True,
         "with_gdi": True,
```

Comment:
gdi and uniscribe are also disabled by default upstream https://github.com/harfbuzz/harfbuzz/blob/2.8.0/CMakeLists.txt#L45-L46

Reply:
yes but disabling gdi in pre build packages might break several downstream packages if they rely on it, if I understand correctly @khaledhosny